src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -123,7 +123,8 @@
     private volatile ResourceResolverFactoryConfig config = DEFAULT_CONFIG;
 
     /** Alias path whitelist */
-    private final List<String> aliasPathAllowList = new CopyOnWriteArrayList<>();
+    @SuppressWarnings("java:S3077")
+    private volatile Set<String> aliasPathAllowList = Collections.unmodifiableSet(Collections.emptySet());
```

Comment:
`Collections.emptySet()` is already immutable. No need to wrap it with `Collections.unmodifiableSet()`.